ChopChop/ViewModels/Users/ProfileViewModel.swift

```diff
@@ -7,46 +7,28 @@ import Combine
 final class ProfileViewModel: ObservableObject {
     /// The id of the user displayed in the profile view.
     private let userId: String
-    /// The ids of the followees of the user.
-    private var followeeIds: [String] = []
+
+    private let storageManager = StorageManager()
+    private let settings: UserSettings
+    @ObservedObject private(set) var recipesViewModel: OnlineRecipeCollectionViewModel
 
     /// User profile details
-    @Published private(set) var userName = ""
+    @Published private(set) var userName = "No name"
```

Comment:
Should we change all the "No name"s to empty strings instead? Because users can't have empty strings as names, but they could name themselves "no name"

Reply:
Will change